src/components/PageDefinitions.js

```diff
@@ -346,15 +342,15 @@ class PageDefinitions extends Component {
     return (
       <DropdownButton
         className="list-button"
-        bsStyle={''}
+        bsStyle="default"
```

Comment:
I had written 'bsStyle={''}' specifically because 'bsStyle="default"' adds unwanted styling we don't want.

bsStyle="default"
<img width="1156" alt="screen shot 2018-07-16 at 10 10 19 am" src="https://user-images.githubusercontent.com/23349235/42772884-95ad0e4c-88e0-11e8-8970-91f9eb2c7c6a.png">
default style adds a box around each filter type, which is not what we want.

bsStyle=''
<img width="1171" alt="screen shot 2018-07-16 at 10 10 57 am" src="https://user-images.githubusercontent.com/23349235/42772944-b73f3788-88e0-11e8-91fa-750cf3e113ce.png">
Having empty strings for bsStyle removes the box.

Reply:
right but putting in the empty string causes an error in the console. We'll need to find a different way to style BS buttons not as buttons.